app/instance-initializers/ember-a11y.js

```diff
@@ -1,10 +1,12 @@
 // This initializer exists only to make sure that the following
 // imports happen before the app boots.
 import Ember from 'ember';
-import { registerKeywords } from 'ember-a11y/ember-internals';
 import pivotRouteIdentifier from 'ember-a11y/utils/pivot-route-identifier';
 
+//htmlbars
```

Comment:
Can we make this look a bit more template-y?

```
// {{htmlbars}}
import { registerKeywords } from 'ember-a11y/ember-internals';
registerKeywords();
// {{/htmlbars}}
```

And for bonus points we can match `{{htmlbars}}.*{{/htmlbars}}` blindly. The comment that doesn't get pulled out will get cleaned up by uglify.